BrainPortal/app/models/group.rb

```diff
@@ -63,6 +63,8 @@ class Group < ActiveRecord::Base
 
   attr_accessible         :name, :description, :site_id, :creator_id, :user_ids
 
+  api_attr_visible        :id, :name, :description, :type, :site_id, :invisible
```

Comment:
Do not show the ;invisible attr (which will eventually be replaced by something better)